src/Components/BadgeIcons.jsx

```diff
@@ -0,0 +1,27 @@
+import Badge from "./Badge";
+import PropTypes from 'prop-types';
+import Icons from "./Icons";
+
+function BadgeIcons(props) {
```

Comment:
what is the use of this file?
Dont you think this is unnecssary here 
fix this